assets/scss/base/_lists.scss

```diff
@@ -1,38 +1,96 @@
+/*
+Lists
 
-/**
-* @desc         lists.
-* @name         layouts
-* @author       Anthony Munene
-* @tested       N/A
-* @requires     N/A
+
+Markup:
+
+<ul>
+  <li class="list-item">
+      <div class="list-item__primary">
+          <h3 class="list-item__heading">Sample heading</h3>
+          <p class="list-item__info">Sample info text</p>
+      </div>
+      <div class="list-item__secondary">
+          <a class="list-item__link list-item__link--right" href="review">CTA link</a>
+      </div>
+  </li>
+</ul>
+
+.list-item                - a list item with a bottom grey border
```

Comment:
`.list-item` feels a little like it is hanging in the wind here, we spoke about suggestions maybe something like `.info-list` on the `ul` then prepend all the selectors with `.info-` @hmrc/assets (open to suggestions on names)